src/ia-bookmark-edit.js

```diff
@@ -52,12 +79,23 @@ export class IABookmarkEdit extends LitElement {
   }
 
   changeColorTo(id) {
-    this.bookmark.color = id;
+    this.newColor = id;
+    this.disableSave = this.shouldDisableSave();
     this.emitColorChangedEvent(id);
   }
 
   updateNote(e) {
-    this.bookmark.note = e.currentTarget.value;
+    this.newNote = e.currentTarget.value;
+    this.disableSave = this.shouldDisableSave();
+  }
+
+  shouldDisableSave() {
+    const colorHasChanged = this.colorStart !== this.newColor;
+    const noteHasChanged = this.noteStart !== this.newNote;
```

Comment:
I'm not sure if this jives with your use case, but if you make these 4 properties reactive, you could make this a getter and its consumers will react to its changes instead of updating `this.disableSave` in different spots above, ie:
```
get disableSave() {
  const colorHasChanged = this.colorStart !== this.newColor;
  const noteHasChanged = this.noteStart !== this.newNote;
  return !colorHasChanged && !noteHasChanged;
}

...

this.disableSave ? 'foo' : 'bar';
```